tests/x11regressions/x11regressions_setup.pm

```diff
@@ -14,19 +14,25 @@ use testapi;
 sub run() {
     my $self = shift;
 
+    # cmd1 grant user permission to access serial port until next reboot
+    # cmd2 get permanent user permission to access serial port even if reboot
+    my $cmd1 = "chown $username /dev/$serialdev";
+    my $cmd2 = "gpasswd -a $username dialout";
+
     x11_start_program("xterm");
 
-    # grant permission for default user to access serial port
-    type_string "xdg-su -c 'chown $username /dev/$serialdev'\n";
-    wait_still_screen;
+    foreach ($cmd1, $cmd2) {
+        type_string "xdg-su -c '$_'\n";
```

Comment:
Why xdg-su for commands ? (xdg-su ? run a GUI program as root after prompting for the root password)